File: C++/19.cpp
#include <iostream>

using namespace std;

int main()
{

    int size, counter = 0, sum = 0;

    cout << "Enter the number of elements: ";
    cin >> size;

    int list[size];

    cout << "Enter elements of the lis: " << endl;

    for (int i = 0; i < size; i++)
        cin >> list[i];

    for (int i = 0; i < size; i++)
    {
        if (list[i] % 2 == 0)
        {
            counter++;
            sum = sum + list[i];
        }
    }

    cout << "Number of the even elements:" << counter << endl;
    cout << "SUm of the even elements:" << sum << endl;
}

File: Python/19.py
myList = []
counter = 0
sum = 0

numElements = int(input("How many elements would you like to add? "))

for i in range(numElements):
    element = int(input("Enter an element: "))
    myList.append(element)

for i in myList:
    if i % 2 == 0:
        sum += i
        counter += 1

print("Sum of the even elements: ",sum)
print("Number of the even elements: ",counter)